index.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
-var Requested = require('./requested')
-  , https = require('https')
-  , http = require('http');
+module.exports = (function () {
+  if ('XMLHttpRequest' in global) {
+    return require('./browser');
+  }
 
-module.exports = Requested.extend({
-  initialize: function initialize() {
+  var XMLHttpRequest = global.XMLHttpRequest = require('node-http-xhr');
```

Comment:
The current issue is that the feature tests rely on a global `XMLHttpRequest` being present anyway; most notably to set `Requests.method` (since `Requests.XHR()` just uses the global `XMLHttpRequest`).